Added photo thumbnails to tweets with entities

rsvargas opened the pull request:
Added an option to show a small preview of photos identified by the entities object. Kinda like Twitter's new interface.

catchin committed:
avoid mentions of similar twitter accounts in own timeline

rsvargas committed:
Correct the position of some fields in the options page

and make some fields larger to improve readability.

rsvargas committed:
Fixed the trending topics API calls.

rsvargas committed:
Added photo thumbnails

and also supporting newlines in tweets with entities.

kav2k committed:
Replace text branding

kav2k committed:
Replace icon branding

kav2k committed:
Wipe API keys

kav2k committed:
Updated the license and readme, v2.0.0.0

kav2k commented:
@rsvargas I'm making a fork of Silver Bird, since it is no longer maintained. Would you mind if I include your patch?

kav2k committed:
Merge pull request #1 from catchin/master

avoid mentions of similar twitter accounts in own timeline

kav2k committed:
Fix lists

kav2k committed:
Icon credit + webstore link

kav2k committed:
Fixed trending places

Fixed desktop notifications (using web notifications)

kav2k committed:
Bump version to 2.0.0.1, fix/bump changelog

kav2k committed:
Use bigger image for notification

rsvargas commented:
Of course I don't mind, feel free to add these. I'll be glad!

kav2k committed:
Magic number fix (tentatively fixes #5)
Code taken from https://github.com/cezarsa/silver_bird/issues/287#issuecomment-55152920

kav2k committed:
Draft work for proper character counting (#6)

theDraco commented:
@kav2k: Do you plan to make it public? What will the name be? Can you make the "mute" and "block" features work? Right now if you block or mute someone, their tweets still show up in Silverbird.

@rsvargas: How do I get this to work? I have copied all 10 files you changed into the 1.9.10.2 Silverbird folder, but when I'm done it says the extension is corrupted. I have done it 3 times now, what am I doing wrong? I'm trying to get images to display in the Silverbird window.

kav2k commented:
@theDraco It's up at https://github.com/kav2k/silver_feather
I changed the name to Silver Feather to avoid confusion. It's currently published to the Web Store but it unlisted (not searchable) until I straighten up more pressing problems. You can install it already as some important fixes are already in.

You can't just replace files of a packed extension, Chrome will block that. You need to download the whole thing and load it as unpacked in developer mode.

theDraco commented:
@kav2k Great, I got it working. A couple questions. Is it possible to get the "mute" and "blocked" functions working in Silverbird? Right now if you mute someone their tweets still show up in Silverbird. The same if you block them.

And one more thing, is it possible to make the image previews full size? Right now if you hover over an image link the preview is very tiny, can I make it bigger?

kav2k commented:
@theDraco I think I'm going to implement larger previews, but it's not high priority. Feel free to add bugs about that and mute/blocked on my issue tracker so I don't forget about it.

kav2k committed:
Read and use Twitter help/configuration data

Fixes #6

kav2k committed:
Bump version to 2.0.0.2. bump changelog

rsvargas commented:
@theDraco I'll probably rewrite this as a pull request do @kav2k 's new fork in a few days. So is probably best to wait for that.

If you still want to get this before and you are having problems with the merge, you can try to checkout this branch from my fork: https://github.com/rsvargas/silver_bird/tree/photo-thumbnails

kav2k committed:
Enable line breaks (#16)
Still needs layout fixes before closing

kav2k commented:
@rsvargas I'm not that eager to merge this in right away; I haven't tested it yet and it's kind of a low priority compared to a lot of [other issues](https://github.com/kav2k/silver_feather/issues). I'd still be happy if you make a pull request - just don't expect it to be merged very fast.

kav2k committed:
Refactored 4889648

kav2k committed:
Layout changes for multiline (closes #16)

kav2k committed:
Bump version to 2.0.0.3, bump changelog

kav2k committed:
Height tweak for c7bd9f6

kav2k committed:
Add Twemoji support (closes #20)

kav2k committed:
Sweep #7 and #9 under the rug for now

rsvargas committed:
Merge remote-tracking branch 'remotes/silver_feather/master' into merge_feather

Conflicts:
	lib/tweet_manager.js
	lib/tweets_assembler.js

rsvargas committed:
Fixes some merge issues with the layout.

rsvargas committed:
Updated translations for 'pt_BR' and 'es'

rsvargas committed:
Removing another merge leftover

kav2k committed:
Switch Twemoji to SVG, exclude common unicode (#20)

kav2k committed:
Tiny options layout fix

kav2k committed:
Make twemoji optional (closes #20)

kav2k committed:
Bump version to 2.0.0.4, bump changelog

Commit:
Added Czech locale

kav2k committed:
Change "both" name format to TDR and make default (#8)

Commit:
Removed not working comments

Comments:
Line 57 //Sjednocený = singular; this is plural
Line 78 // hledání -frequent
Line 81 // od (person); z (place)
Line 87 //sent to user - odesláno uživateli
Line 114  //Odpověď - noun; Odpověďet - verb
Line 117  //verb: Retweetnout
Line 370  //1 minuta, 2 minuty, 3 minuty, 4 minuty, 5 minuty, 6 minuty, 7 minut ...
Line 387  //same as minutes
Line 402  // we do not use X years ago. we use 'před X lety' - something like before X years
Line 441  //Where? Nahoře
Line 444  //Where? Dole

kav2k committed:
Merge branch 'master' of https://github.com/Shifterovich/silver_feather into Shifterovich-master

kav2k committed:
Merge branch 'Shifterovich-master'

kav2k committed:
Twemoji update, bug workaround

kav2k committed:
Add support for displaying quoted tweets #26

Including: custom color, auto-expand

rsvargas committed:
Merge branch 'master' of https://github.com/kav2k/silver_feather into merge_feather

Conflicts:
	css/options.css
